008_team_6_agent_design/008_team_6_agent_design.tex

```diff
@@ -1 +1,1021 @@
+
+
+%%%%%%%%%%
+%%%%%%%%%% TRUST
+%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
+\section{Formation of Social Network through Trust}\label{trust}
+
+\subsection{Background}
+
+With treaties offering a notion of socially accepted institutional power, the formation of treaties must be handled carefully, so as not to disadvantage the agents who engage in them. For this reason, a social network must be constructed to offer a heuristic for joining into these agreements. 
+
+Trust serves as ``an important role for the formation of coalitions in social networks and in determining how high value of information flows through the network'' \cite{trustEEE}, with Ostrom futhermore supporting that ``trust is an essential social lubricant'' \cite{ostrom2003trust}. Ostrom further asserts that trust ``affects the willingness to cooperate'' \cite{ostrom2003trust} and ``enhances cooperation in social-dilemma experiments'' \cite{ostrom2003trust}, providing clear reasoning that trust is an effective parameter to introduce to the agent structure.
+
+Finally, it is through this introduction of trust that we demonstrate the assertion from Adali's work \cite{trustEEE} that trust facilitates ``communication behaviors which are statistically different from random communications''.
+
+\subsection{Initialisation}
+The agent takes a simple algorithmic approach to formulating a social network; firstly, the ID of the neighbouring agents is determined through message passing, this ID is mapped to an initial trust value and the trust value is continually updated through subsequent social interactions. 
+
+This agent implementation offers different initialised trust values based on the social motive, using the general notion that the more narcissistic an agent is, the less likely it is to have an initial positive opinion of their neighbours. 
+
+For this reason, we give a maximal range of trust from -25 to +25 with the initialisations as follows:
+
+\begin{itemize}
+    \item Narcissist: -5
+    \item Selfish: 0
+    \item Collectivist: +5
+    \item Altruist: +10
+\end{itemize}
+
+\subsection{Causes for Trust Update}
+
+It is through continuous social interaction in the tower that agents will be able to form opinions of the other agents. For this agent, there is a specific set of interactions that will result in a opportunity for trust to update. These interactions impose a \textit{base} behavioural update, before scaling due to social motive (\Cref{sec:varTrust}).
+
+\subsubsection{Treaties}
+
+\begin{enumerate}
+    \item When an agent receives a response to the treaty it has proposed: 
+    \begin{itemize}
+        \item If accepted: +3
+        \item If rejected: -2
+    \end{itemize}
+    \item When an agent receives a treaty proposal: 
+    \begin{itemize}
+        \item  If the treaty is evaluated as good: +2
+        \item   If the treaty is evaluated as bad: -1
```

Comment:
What do these values actually mean? Is it the base behavioural update? Should phrase it more clearly, I'm still confused.